ReleaseHistory.md

```diff
@@ -16,6 +16,7 @@
 - NEW => new feature 
 
 ## UNRELEASED
+* BUG: Fix `TryGetPortablePdbMetadataReader` unexpectedly causes `UnauthorizedAccessException` error when the PDB file is missing. [1001](https://github.com/microsoft/binskim/pull/1001).
```

Comment:
>TryGetPortablePdbMetadataReader [](http://example.com/codeflow?start=12&length=31)

`due to evaluating to Pdb.PdbLocation being populated with the load directory (not an existing location).`